spec/models/specification_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
+
+RSpec.describe Specification, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
Thanks @andela-iarisah, the tests meant for for this file were unconsciously overlooked. They would be added as soon as I finish this message.